modules/modules.nix

```diff
@@ -34,7 +34,9 @@ let
     ./misc/numlock.nix
     ./misc/pam.nix
     ./misc/qt.nix
+    ./misc/qt/kconfig.nix
     ./misc/specialisation.nix
+    ./misc/qt/kconfig.nix
```

Comment:
I think this path got included twice when resolving a merge conflict somewhere.

Came across this PR while trying to setup my home manager config after switching to KDE for Plasma 6, looking forward to seeing it merged!

Reply:
Fixed